source/content/terminus/02-install.md

```diff
@@ -83,68 +51,70 @@ If you are a developer contributoring to Terminus, you can install it from its [
   - You can check your OpenSSH version by running `ssh -V` from a terminal application.
   - This package is required for executing nested Drush or WP-CLI commands.
 
-### Incompatible Operating Systems
-
-Terminus does not work with the following platforms:
-
-- Windows 10 – Command Line
-- Windows 10 – Git Bash (MingW)
-- Ubuntu 18.0 and earlier versions
-- Linux system with coreutils equal to or less than 8.28
-
-## Authenticating with Terminus
-
-You must log in with a machine token after the installation completes. A machine token is used to securely authenticate your machine. Machine tokens provide the same access as your username and password, and do not expire. Refer to [Machine Tokens](/machine-tokens/) for more information.
+## Installation and Update Methods 
+### MacOS Terminus Installation
```

Comment:
We could make headers even more simple. Just callout MacOS vs Windows/Linux and get rid of the h3s for updates: 
<img width="784" alt="Screenshot 2025-01-30 at 2 19 07 PM" src="https://github.com/user-attachments/assets/4e62af3f-c769-4f4b-a246-17400f90f8ed" />

This is my preference but I figured it's better to discuss first before imposing it in the PR